external-crates/move/crates/move-analyzer/editors/code/scripts/create.sh

```diff
@@ -64,29 +63,29 @@ rm -rf $LANG_SERVER_DIR
 mkdir $LANG_SERVER_DIR
 
 for DIST_OS VSCODE_OS in "${(@kv)SUPPORTED_OS}"; do
-    # Iota distribution identifier
-    IOTA_DISTRO=$NETWORK"-v"$VERSION
-    # name of the Iota distribution archive file, for example iota-testnet-v1.0.0-macos-arm64.tgz
-    IOTA_ARCHIVE="iota-"$IOTA_DISTRO"-"$DIST_OS".tgz"
+    # IOTA distribution identifier
+    IOTA_VERSION="v"$VERSION
```

Comment:
Just a preference?

Reply:
The name of the released files changed, didn't work without this change